.github/workflows/release_to_stable.yaml

```diff
@@ -0,0 +1,136 @@
+          echo "applications:" > overlay.yaml
+          echo "  legend-db:" >> overlay.yaml
+          echo "    channel: ${{ steps.release-short.outputs.yyyy_mm }}/stable" >> overlay.yaml
+          echo "  legend-sdlc:" >> overlay.yaml
+          echo "    channel: ${{ steps.release-short.outputs.yyyy_mm }}/stable" >> overlay.yaml
+          echo "  legend-engine:" >> overlay.yaml
+          echo "    channel: ${{ steps.release-short.outputs.yyyy_mm }}/stable" >> overlay.yaml
+          echo "  legend-studio:" >> overlay.yaml
+          echo "    channel: ${{ steps.release-short.outputs.yyyy_mm }}/stable" >> overlay.yaml
+          echo "  gitlab-integrator:" >> overlay.yaml
+          echo "    channel: ${{ steps.release-short.outputs.yyyy_mm }}/stable" >> overlay.yaml
+
+          # We can't have bundle.yaml as both the source and destionation, the result will not be
+          # as we would have wanted it.
+          cp bundle.yaml bundle2.yaml
+          yq eval-all '. as $item ireduce ({}; . *+ $item)' bundle2.yaml overlay.yaml > bundle.yaml
+
+          cat bundle.yaml
+
+      - name: Create branch and commit for bundle
+        uses: EndBug/add-and-commit@v9 # You can change this to use a specific version.
+        with:
+          # The arguments for the `git add` command (see the paragraph below for more info)
+          # Default: '.'
+          add: "bundle.yaml"
+
+          # The name of the user that will be displayed as the author of the commit.
+          # Default: depends on the default_author input
+          author_name: Sergiu Miclea
+
+          # The email of the user that will be displayed as the author of the commit.
+          # Default: depends on the default_author input
+          author_email: smiclea@cloudbasesolutions.com
+
+          # Additional arguments for the git commit command. The --message argument is already set by the message input.
+          # Default: ''
+          commit: --signoff
+
+          # The message for the commit.
+          # Default: 'Commit from GitHub Actions (name of the workflow)'
+          message: "Create ${{ github.event.inputs.release }}/stable release"
+
+          # If this input is set, the action will push the commit to a new branch with this name.
+          # Default: ''
+          new_branch: "release-${{ github.event.inputs.release }}-stable"
+
+      - name: Upload bundle to stable
+        run: |
+          charmcraft pack
+          charmcraft whoami
+          charmcraft upload finos-legend-bundle.zip --verbose
+
+          release_short="${{ steps.release-short.outputs.yyyy_mm }}"
+
+          # Get the bundle revision for the specified release
+          bundle_status=$(charmcraft status --format json finos-legend-bundle)
+          release_info=$(echo "$bundle_status" | jq -r '.[].mappings[].releases[] | select(.channel == "'"$release_short"'/edge")')
+          bundle_rev=$(echo "$release_info" | jq -r '.revision')
```

Comment:
Basically, you're creating a new bundle, uploading it, and then release the edge one as stable instead, which would still deploy the edge charms.

Also, below, if you don't have the ``release_info``, we do know the bundle revision, it's the latest that was uploaded just a few seconds ago at L121.

Basically, all we need to do here is:

```
          # Get the last revision number and release.
          bundle_rev=$(charmcraft revisions finos-legend-bundle | awk 'FNR == 2 {print $1}')
          charmcraft release finos-legend-bundle --revision=${bundle_rev} --channel="${{ steps.release-short.outputs.yyyy_mm }}/stable" --verbose
```

Reply:
I have added the latest bundle revision as the revision for the release. 

You said that I'm releasing the edge one as stable, but at L95 I'm adding the latest bundle.yaml which contains the stable charms overlayed at L86. Doesn't this mean that I'm actually releasing the stable charms? What exactly do I need to do differently?

Reply:
So, in the first steps we're releasing the exact same charm edge revisions to stable, releasing them. A bundle is basically just a yaml file in which we describe a set of charms to be deployed, including the charms' branches as well. Basically, you could say a stable bundle deploys stable charms, hence why we needed the overlay above, and why we need to upload the new bundle and release it to stable.

Right now, this seems to be what we need.